### PR TITLE
introduce forecast proposal status media type

### LINKED ISSUE #74: deprecate "missing" forecast endpoints, implement proposal status response

### PR DESCRIPTION
**Pull request by caindy**
This has a number of implications discussed in and closes #74

### CODE REVIEW

File: docs/_data/components/schemas/array-max-monitored-elements.yaml [hunk 1]
```diff
@@ -44,11 +44,84 @@ forecast-proposal:
...
+      $ref: ./data-provenance.yaml
+    incomplete-obligation-count:
+      description: |
+
+        The number of facilities for this provider whose Ratings Obligation has
+        not been met in this forecast window. This number may be larger than the
+        size of `incomplete-facilities`, since the latter has a pre-defined
+        upper bound for performance and application security reasons.
+ 
+        ⚠️ **The Ratings Provider should assert that this value is zero when they
```
Comment (catkins-miso) [inline docs/_data/components/schemas/array-max-monitored-elements.yaml]:
```suggestion
        ⚠️ **The Ratings Provider should check that this value is zero when they
```

**Review by catkins-miso — CHANGES_REQUESTED** [review]

**Review by catkins-miso — APPROVED** [review]

File: docs/_data/paths/rating-proposals_forecasts.yaml [hunk 2]
```diff
@@ -1,44 +1,38 @@
...
 
-get:
-  <<: *head
-  operationId: getRatingForecastProposals
-  description: >
-    Obtain the Ratings Forecasts that *this* Ratings Provider has submitted.  
-    Note that the offset-period-start and period-end parameters work in a
-    similar fashion to  getLimitsForecastSnapshot, except that it will return
-    any proposal that overlaps with the implied period.  
+    Note that the same status object is returned in each
```
Comment (catkins-miso) [inline docs/_data/paths/rating-proposals_forecasts.yaml]:
```suggestion
    Note that the same status object is returned for each
```

File: docs/_data/paths/rating-proposals_forecasts.yaml [hunk 3]
```diff
@@ -1,44 +1,38 @@
...
+    `patchRatingForecastProposal`, so this endpoint may seem redundant. However,
+    an anticipated use cases for this `GET` endpoint are to allow supervisor
```
Comment (catkins-miso) [inline docs/_data/paths/rating-proposals_forecasts.yaml]:
```suggestion
    an anticipated use case for this `GET` endpoint is to support supervisor
```

File: docs/_data/paths/rating-proposals_forecasts.yaml [hunk 4]
```diff
@@ -1,44 +1,38 @@
...
+    processes that are setup to independently ensure the provider's program for
```
Comment (catkins-miso) [inline docs/_data/paths/rating-proposals_forecasts.yaml]:
```suggestion
    processes that are setup by the client to independently ensure the provider's process for
```

**Review by getorymckeag — APPROVED** [review]
OK, looks good.  Just to be clear, while I don't have any critical comments, I did some due diligence with our devs on this one.  I think it solves a number of problems well.

**Pull request merged**